driver-core/src/test/java/com/datastax/driver/core/PrimitiveTypeSamples.java

```diff
@@ -56,6 +56,8 @@
                 .put(DataType.cint(), Integer.MAX_VALUE)
                 .put(DataType.text(), "text")
                 .put(DataType.timestamp(), new Date(872835240000L))
+                .put(DataType.date(), -2147467577)
```

Comment:
UDT definition by column is defined as:


    a ascii
    b bigint
    c blob
    d boolean
    e decimal
    f double
    g float
    h inet
    i int
    j text
    k timestamp
    l uuid
    m varint
    n timeuuid
    o 'org.apache.cassandra.db.marshal.SimpleDateType'
    p time
    q smallint
    r tinyint

so everything seems fine except for column o which should be 'date'.